Assets/Scripts/MusicManager.cs

```diff
@@ -0,0 +1,75 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+[RequireComponent(typeof(AudioSource))]
+[RequireComponent(typeof(AudioLowPassFilter))]
+public class MusicManager : MonoBehaviour
+{
+    [SerializeField]
+    private AudioSource music;
+    float lastTime;
+    const float MIN_FREQ = 500;
+    const float MAX_FREQ = 15000;
+
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        GetComponent<AudioLowPassFilter>().cutoffFrequency = MIN_FREQ;
```

Comment:
Rather than keep calling GetComponent, I would save the AudioLowPassFilter to a variable of this class and then you can just use that instead